### ISSUE TITLE
Add ffmpeg argument to scale video that the dimensions are always divisible by 2

### ISSUE THREAD

**Issue by jamesrobinsonvfx** [jamesrobinsonvfx/mplay_batch#10]
If flipbook image dimensions are not divisible by 2, ffmpeg will complain and fail. The error is not reported to the user though, so it can be hard to debug. 

Add the following to force good dimensions:

```
-vf "pad=ceil(iw/2)*2:ceil(ih/2)*2"
```

**Comment by jamesrobinsonvfx** [comment]
or
```
-vf "crop=trunc(iw/2)*2:trunc(ih/2)*2"
```

**Comment by jamesrobinsonvfx** [comment]
Wound up going with `pad`. Otherwise, you end up with a single black pixel border on one side with `pad`

**Comment by jamesrobinsonvfx** [comment]
Fixed in #11
